domains/sbdesigns.json

```
@@ -4,7 +4,13 @@
     "email": "sebandblocks@turquoisetnt.one"
   },
   "records": {
-    "CNAME": "sbdesigns.pages.dev"
+    "CNAME": "sbdesigns.pages.dev",
+    "MX": [
+      { "target": "mx.zoho.eu", "priority": 10 },
+      { "target": "mx2.example.com", "priority": 20 },
+      { "target": "mx3.example.com", "priority": 50 }
+    ],
+    "TXT": "v=spf1 include:zohomail.eu ~all"
   },
   "proxied": false
```

```suggestion
  "proxied": true
```

You can't mix and match CNAME with other records, unless the proxied is true